config/software/inspec.rb

```diff
@@ -17,11 +17,15 @@
 name "inspec"
 default_version "master"
 
+license "Apache-2.0"
+license_file "LICENSE"
+
 source git: "https://github.com/chef/inspec.git"
 
 dependency "ruby"
 dependency "rubygems"
 dependency "bundler"
+dependency "nokogiri"
```

Comment:
Yes: without this, it will be dragged into the build by the [bundle install call](https://github.com/chef/omnibus-software/pull/759/files#diff-55cf07643f42fcc9090c5a14f4126db4R36), and end up failing health checks on rhel-7 (only).